aiohue/lights.py

```diff
@@ -90,6 +90,25 @@ def colorgamut(self):
 
         return color_gamut
 
+    def process_update_event(self, update):
+        state = dict(self.state)
+
+        if (color := update.get("color")) :
+            state["xy"] = [color["xy"]["x"], color["xy"]["y"]]
+
+        if ct := update.get("color_temperature"):
+            state["ct"] = ct["mirek"]
+
+        if "on" in update:
+            state["on"] = update["on"]["on"]
+
+        if dimming := update.get("dimming"):
+            state["bri"] = dimming["brightness"]
```

Comment:
Update events have their own format.

These are the ones that I found so far for lights.